### ISSUE TITLE
Plus button does nothing in discovery mode

### ISSUE THREAD

**Issue by snelltheta** [bottiger/SoundWaves#60]
Yes, it makes a cool transition to an x but doesn't do anything else.

![tmp_5339-screenshot_2015-09-14-19-32-181995746261](https://cloud.githubusercontent.com/assets/12014328/9866473/937bd9d6-5b17-11e5-960a-1c8146dba5e7.png)

**Comment by bottiger** [comment]
@snelltheta Yeah - that part of the code is in for a big rewrite soon. But it will probably take a week or two before I get to it since it do not regard it as critical
